tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -68,6 +68,22 @@ spec:
         N=$(params.concurrentLimit)  # The maximum number of images to be processed at once
         count=0
         COMPONENTS_LENGTH=$(jq '.components |length' ${SNAPSHOT_PATH})
+
+        # Add a function to check if the image is already signed
+        function is_signed() {
+          local registry_reference=$1
+          local tag=$2
+          local manifest_digest=$3
+
+          # Check if the signature exists using skopeo
+          if skopeo inspect --raw "docker://${registry_reference}:${tag}" \
+            | jq -e ".signatures | any(.digest == \"${manifest_digest}\")" > /dev/null; then
```

Comment:
How about removing the redirection of stdout? It will basically just print "true" or "false" and I think there is no harm in that, right? I think that could actually be beneficial when debugging.

Reply:
Yes, that makes sense; thanks!